ptrack-gentest/gentests.sh

```diff
@@ -0,0 +1,45 @@
+C=$#
```

Comment:
Script is missing
#!/bin/bash or #!/bin/sh

Reply:
corrected